### PR TITLE
:innocent: Return no more than 3 nearby results

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

**Comment by st3v3nhunt** [comment]
[Remove 4th nearby result if the open one is not one of them](https://trello.com/c/gxhJitJX/628-remove-4th-nearby-result-if-the-open-one-is-not-one-of-them)

File: test/unit/lib/dedupePharmacies.js [hunk 1]
```diff
@@ -48,6 +48,28 @@ describe('dedupePharmacies', () => {
...
+          { identifier: 'AAA02' },
+          { identifier: 'AAA03' },
+          { identifier: fourthId },
+        ] };
+
+      const result = dedupe(services);
+
+      expect(result.open.length).is.equal(1);
+      expect(result.nearby.length).is.equal(3);
+      result.nearby.forEach((item) => {
```
Comment (neilbmclaughlin) [inline test/unit/lib/dedupePharmacies.js]: There are collection expectations contains and includes which I'm guessing can be negated. (http://chaijs.com/api/bdd/)

Comment (st3v3nhunt) [inline test/unit/lib/dedupePharmacies.js]: Care to provide an example?

Comment (neilbmclaughlin) [inline test/unit/lib/dedupePharmacies.js]: Not at my machine but as a guess

expect(nearby.map((p) => p.identifier)).to.not.include(fourthId)

File: test/unit/lib/dedupePharmacies.js [hunk 2]
```diff
@@ -4,47 +4,58 @@ const chai = require('chai');
...
+      };
 
       const result = dedupe(services);
 
       expect(result.open.length).is.equal(0);
       expect(result.nearby.length).is.equal(3);
-      result.nearby.forEach((item) => {
-        expect(item.identifier).to.not.equal(fourthId);
-      });
+      expect(result.nearby).to.deep.not.include.members([fourthItem]);
```
Comment (neilbmclaughlin) [inline test/unit/lib/dedupePharmacies.js]: Just an observation but it's concise and does the job but it doesn't read well.

**Pull request merged**